modules/react/package.json

```diff
@@ -38,13 +38,12 @@
     "@types/styled-react-modal": "^1.2.5",
     "boxicons": "^2.1.4",
     "prop-types": "^15.8.1",
-    "styled-components": "^4.4.1",
+    "styled-components": "^6.1.1",
     "styled-react-modal": "^3.1.1",
     "supercluster": "^8.0.1"
   },
   "peerDependencies": {
-    "react": "^16.14 || ^17",
-    "react-dom": "^16.14 || ^17"
+    "react": "^18"
```

Comment:
Let's follow deck.gl and use "^18.2.0": https://github.com/visgl/deck.gl/blob/master/modules/react/package.json#L42.

Reply:
I changed it to `^18.2.0`. Also updated to examples, that were still requesting react 16.